mercury-operator/install/mercury-operator.yml

```diff
@@ -86,6 +86,45 @@ rules:
       - customresourcedefinitions
     verbs:
       - "*"
+  - apiGroups:
```

Comment:
you can add more apigroups to the same rule as long as they share the same verbs:
```
- apigroups:
    - ""
    - mercury.redhat.io
    - kafka.strimzi.io
    - camel.apache.org
```

Reply:
Like this ?
- apigroups:
    - ""
    - mercury.redhat.io
    - kafka.strimzi.io
    - camel.apache.org
    resources:
      - kafkas
      - kafkatopics
      - serviceaccounts
      - services
      - integrations
      - servicedomainclusters
      - servicedomainclusters/status
      - servicedomains
      - servicedomains/status
    verbs:
      - "*"